BENCHMARK_VERIFICATION.md

```
@@ -0,0 +1,98 @@
+git checkout <commit-before-upgrade>
+
+# Build and run the deprecated benchmark
+cargo run --release --example bench_deprecated --features deprecated_api
+```
+
+You should see output like:
+```
+    Rows |      NNZ | Deprecated (us)
+---------+----------+----------------
+     100 |      979 |          56.00
+    1000 |    10037 |         487.00
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)